composer.json

```diff
@@ -4,49 +4,58 @@
     "type": "project",
     "description": "Symfony.se",
     "autoload": {
-        "psr-0": { "": "src/" }
+        "psr-4": {
+            "": "src/"
+        },
+        "classmap": [
+            "app/AppKernel.php",
+            "app/AppCache.php"
+        ]
     },
+
     "require": {
-        "php": ">=5.3.3",
-        "symfony/symfony": "2.3.*",
-        "doctrine/orm": ">=2.2.3,<2.4-dev",
-        "doctrine/doctrine-bundle": "1.2.*",
-        "twig/extensions": "1.0.*",
-        "symfony/assetic-bundle": "2.3.*",
-        "symfony/swiftmailer-bundle": "2.3.*",
-        "symfony/monolog-bundle": "2.3.*",
-        "sensio/distribution-bundle": "2.3.*",
-        "sensio/framework-extra-bundle": "2.3.*",
-        "incenteev/composer-parameter-handler": "~2.0",
-        "knplabs/knp-markdown-bundle": "1.2.*",
-        "dms/meetup-api-bundle": "~1.0",
-        "happyr/slugify-bundle": "1.0.*",
-        "debril/rss-atom-bundle": "~1.3",
-        "happyr/excerpt-bundle": "0.1.*"
+        "php": ">=5.3.9",
+        "symfony/symfony": "2.8.*",
+        "doctrine/orm": "^2.4.8",
+        "doctrine/doctrine-bundle": "^1.4",
+        "twig/twig": "^1.0||^2.0",
```

Comment:
No, we don't have to support both. We should update it to the latest if we want to switch to Symfony3.  If there is some deprecation warning or BC break by using version 1.0. I can remove it. It was working for me on localhost.